zerver/openapi/zulip.yaml

```diff
@@ -12750,6 +12750,57 @@ paths:
                     description: |
                       An example JSON error response for when the public data export
                       exceeds the maximum allowed data export size.
+  /export/realm/consents:
+    get:
+      operationId: get-realm-export-consents
+      summary: Get all private data export consents
```

Comment:
This summary feels overly wordy. I think probably "Get data export consent state" would be a bit more clear and concise.